temel_bilgiler/giriş/web-geliştirmeye-giriş.md

```diff
@@ -0,0 +1,97 @@
+### Giriş
+
+Web geliştiricinin görevi nedir? Bir bakıma, websitesi yapma ve onun bakımından sorumludurlar.
+
+Web geliştiriciler genelde ürününü veya hizmetini web'e taşımaya çalışan müşteriler için çalışırlar. Çalışma genelde proje odaklıdır ve müşterinin ihtiyaçlarını son ürüne koordine eden bir ekip ile işbirliği yapmayı içerir. Müşteri bir teknoloji şirketi, bir organizasyon veya bir hükümet olabilir. Çalışma Frontend, Backend veya FullStack web geliştirme içerebilir.
+
+Web geliştirme eğer mantıksal problemleri çözmeyi seviyor, yeni kullanışlı şeyler yapmayı seviyor ve yeni teknolojileri denemeyi seviyorsanız sizin için iyi bir meslek olabilir. Web geliştiriciler genelde iyi bir iş/yaşam dengesine sahiptir, diğer iş alanlarına göre daha dolgun maaş alırlar ve yüksek talep görürler. Web geliştirme iş fırsatlarınızı daha iyi anlamak için özel konumunuzu Google'da aratın.
```

Comment:
Eğer mantıksal problemleri çözmeyi, yararlı şeyler inşa etmeyi ve yeni teknolojiler deneyimlemeyi seviyorsanız; web geliştirme sizin için iyi bir meslek olabilir. Web geliştiriciler genelde iyi bir iş-yaşam dengesine sahiptir, dolgun maaş alırlar ve yüksek talep görürler. Yaşadığınız yerdeki web geliştirme işi fırsatları hakkında daha iyi fikir sahibi olmak için Google'da arama yapın.